ethereum/spec/src/main/java/tech/pegasys/teku/spec/logic/common/helpers/MiscHelpers.java

```diff
@@ -186,17 +190,25 @@ public void shuffleList(int[] input, Bytes32 seed) {
           flip = pivot - i;
           bitIndex = i & 0xff;
           if (bitIndex == 0 || i == mirror1) {
-            hashBytes = Hash.sha256(seed, roundAsByte, uintToBytes(i / 256, 4));
+            seed.update(digest);
+            roundAsByte.update(digest);
+            uintToBytes(i / 256, 4).update(digest);
+
+            hashBytes = digest.digest();
```

Comment:
these 2 blocks are basically still the sha256.. not against having our own implementation, but can we wrap it in a fn so it's still relatively easy to read?